frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +245,26 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbs = (modelRef, kindObj, match) => {
```

Comment:
Why not just take the `model` instead of `modelRef`? We should know it in all cases this is called, correct?

As a rule, we should avoid `modelFor` when possible since it only works with static models.

Reply:
Is there another method I should call instead of `modelFor`?

Reply:
Don't you already know the model in all cases where this is called? Can you just pass `ClusterRoleBindingModel` instead of passing the name and having to find it with `modelFor`?

Reply:
No, for `ClusterRoleBindingModel` we really want to use the breadcrumb for `RoleBinding` Model.   Same for `ClusterRoleModel` we really want to use breadcrumb for `Role` model.

Reply:
I didn't realize the page was reused for both resources. Disregard my original comment.